core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.scss

```diff
@@ -19,9 +19,10 @@ dot-whats-changed,
 }
 
 .dot-edit__page-wrapper {
+    background-color: $gray-bg;
     height: 100%;
     max-width: 100%;
-    min-width: 100%;
+    padding: $spacing-5;
```

Comment:
Can you ask Freddy about this? He removed that `spacing` variable in this PR: https://github.com/dotCMS/core/pull/23608

I'm not sure if the right thing to do is keep it or delete it.